experiments/EclipseExample/src/test/java/edu/washington/cs/dt/impact/ExampleDT/CalculatorTestCase2.java

```diff
@@ -0,0 +1,71 @@
+package edu.washington.cs.dt.impact.ExampleDT;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import edu.washington.cs.dt.impact.ExampleDT.Calculator;
+
+public class CalculatorTestCase2 {
```

Comment:
CalculatorTestCase2What are the differences between this and CalculatorTestCase1?